dapps/marketplace/src/components/Dropdown.js

```diff
@@ -49,10 +49,23 @@ class Dropdown extends Component {
         onMouseOut={() => (this.mouseOver = false)}
       >
         {this.props.children}
-        {this.props.content && this.props.open ? this.props.content : null}
```

Comment:
This is a fairly major change as now all dropdown content will be rendered and hidden with CSS regardless if the dropdown is open... in a lot of cases that will mean loading extra data that was not required before. Why was this change needed?

Reply:
I added CSS animation for slide in/out for `.dropdown-menu`. The animations didn't happen if the element was rendered instantly with the class `.show`. 

Didn't think about it this way. Will change it.